README.md

```diff
@@ -73,31 +73,53 @@ Dotnet pre-compiled templates are currently hosted by the following clients. Ple
 4. Each of the directory within the tags directory represent a runtime, for testing runtime `v4` templates open directory `v4`
 5. Open the `LastKnownGood` file in the directory for the runtime version you want to test and note the release version present in the file
 6. Open the templates output directory, `..\bin\VS`
-    1. Rename `Microsoft.Azure.WebJobs.ItemTemplates.X.0.0.nupkg` to `ItemTemplates.nupkg`
-    2. Rename `Microsoft.Azure.WebJobs.ProjectTemplates.X.0.0.nupkg` to `ProjectTemplates.nupkg`
-  > To test dotnet-isolated, rename `Microsoft.Azure.Functions.Worker.ItemTemplates.X.0.0` and `Microsoft.Azure.Functions.Worker.ProjectTemplates.X.0.0` in above step.
+    > To test dotnet-inproc, copy `Microsoft.Azure.WebJobs.ItemTemplates.X.X.X` and `Microsoft.Azure.WebJobs.ProjectTemplates.X.X.X`.
+  
+    > To test dotnet-isolated, copy `Microsoft.Azure.Functions.Worker.ItemTemplates.X.0.0` and `Microsoft.Azure.Functions.Worker.ProjectTemplates.X.0.0`.
 7. Open the templates cache directory for release version matching the one found in step 5: `%userprofile%\AppData\Local\AzureFunctionsTools\Releases\<releaseVersion>` 
 8. Open the `templates` folder for the framework you want to test:
-    1. For in-proc, use the `templates` folder fould at the root of the templates cache directory
+    1. For in-proc, use the `templates` folder found at the root of the templates cache directory
     2. For net7-isolated, use the `net7-isolated/templates` folder (for isolated, you should see a folder for netfx, net6, net5 etc.)
-9. Replace the contents of the folder with the renamed package found in `..\bin\VS`
+9.  Replace the contents of the folder with the copied packages found in `..\bin\VS`

```

Comment:
```suggestion
9. Replace the contents of the folder with the copied packages found in `..\bin\VS`

```

Reply:
So we just need to copy the template packages over and don't need to rename them? That's good to know!